lib/horoshop.rb

```diff
@@ -10,7 +14,35 @@ def initialize(url:, username:, password:)
     @expiration_timestamp = nil
   end
 
+  def setup_connection
+    @conn = Faraday.new(url: @url) do |faraday|
```

Comment:
This properties should be separated into a single class. We will send request from other places too, but HTTP parameters always will be the same. This is allowing us to write more clear code.